libraries/RC_Channel/AP_RCSwitch.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include <stdint.h>
+
+class AP_RCSwitch {
+public:
+
+    void init();
+    virtual void read_aux_all();
+
+    // Aux Switch enumeration
+    enum aux_func {
+        DO_NOTHING =           0, // aux switch disabled
+        FLIP =                 2, // flip
+        SIMPLE_MODE =          3, // change to simple mode
+        RTL =                  4, // change to RTL flight mode
```

Comment:
We don't run into problems or risk by having RTL within this enum and other enums as well?